### PR TITLE
added support for forwardOriginalToken

### PR DESCRIPTION
**Pull request by kevin-jenson**
Currently working on Containerizing the CDP Experience API and with the help of Derek was able to get it deployed and was seeing that the Authorization header is being stripped from the request by istio looking at their docs it seems that we need to add a `forwardOriginalToken` key which is set to false by default and omits the token from the request.

istio docs where i found this: https://istio.io/docs/reference/config/security/jwt/

also not 100% sure how you do your versioning so i just incremented patch version by 1?? but i can obviously change it to what ever thanks!

### CODE REVIEW

File: charts/service/templates/istio-policy.yaml [hunk 1]
```diff
@@ -13,6 +13,7 @@ spec:
   jwtRules:
   - issuer: {{ $auth.issuer }}
     jwksUri: {{ $auth.jwks }}
+    forwardOriginalToken: {{ $auth.forwardOriginalToken }}
```
Comment (paraplexed) [inline charts/service/templates/istio-policy.yaml:16]: Let's add a default to the values.yaml -- I think we want this defaulted to true (it also helps with documentation purposes).

Comment (fuzzybaird) [reply charts/service/templates/istio-policy.yaml:16]: I already approved, but just so I don't have to come back to it when you want to merge it and have the default value in.

**Comment by fuzzybaird** [comment]
I have updated the version on this PR so it is primed to be the next merge.

**Comment by fuzzybaird** [comment]
> Currently working on Containerizing the CDP Experience API and with the help of Derek was able to get it deployed and was seeing that the Authorization header is being stripped from the request by istio looking at their docs it seems that we need to add a `forwardOriginalToken` key which is set to false by default and omits the token from the request.
> 
> istio docs where i found this: https://istio.io/docs/reference/config/security/jwt/
> 
> also not 100% sure how you do your versioning so i just incremented patch version by 1?? but i can obviously change it to what ever thanks!

@kevin-jenson yah it is a little confusing. The Chart version needs to be 1 higher than the one in master, in order to build this particular charts next versioned release. And if PR's get merged before yours you have to check master and update the chart version again. I wanted to get the pipeline to use the build numbers to do this, but it is a little tricky as there are many charts in this repo, that need to be at different levels of versioning.

**Pull request merged**